### PR TITLE
tailcfg, control/controlclient: add MapResponse.PingRequestURL

### PR DESCRIPTION
**Pull request by bradfitz**
So the control server can test whether a client's actually present.

Most clients are over HTTP/2, so these pings (to the same host) are
super cheap.

This mimics the earlier goroutine dump mechanism.

### CODE REVIEW

File: control/controlclient/direct.go [hunk 1]
```diff
@@ -1204,3 +1208,22 @@ func ipForwardingBroken(routes []netaddr.IPPrefix) bool {
...
+		log.Printf("answerPing: %v", err)
+		return
+	}
+	t0 := time.Now()
+	_, err = c.Do(req)
+	d := time.Since(t0).Round(time.Millisecond)
+	if err != nil {
+		log.Printf("answerPing error: %v to %v (after %v)", err, targetURL, d)
+	} else {
+		log.Printf("answerPing complete to %v (after %v)", targetURL, d)
```
Comment (crawshaw) [inline control/controlclient/direct.go:1227]: If we want to use this as a regular part of the protocol we'll want to not log in typical happy case. Maybe the ping request should signal whether or not to log? Maybe KeepAlive==true => no log on success?

**Review by crawshaw — COMMENTED** [review]
Good idea!

File: control/controlclient/direct.go (hunk 1 above)
Comment (bradfitz) [reply control/controlclient/direct.go:1227]: I can make a new PingRequest struct. A bit overkill considering how much we currently log on success, though. :)

But yeah, we may want control over it to intentionally put stuff in the logs during debugging.

**Comment by bradfitz** [comment]
PTAL

**Review by crawshaw — APPROVED** [review]

**Pull request merged**